docs/writing-policies/metadata.md

```diff
@@ -0,0 +1,99 @@
+  io.kubewarden.policy.ociUrl: ghcr.io/myorg/my-policy
+  # kubewarden specific:
+  io.kubewarden.policy.title: My policy
+  io.kubewarden.policy.description: Short description
+  io.kubewarden.policy.author: myself
+  io.kubewarden.policy.url: https://github.com/yourorg/my-policy
+  io.kubewarden.policy.source: https://github.com/yourorg/my-policy
+  io.kubewarden.policy.license: Apache-2.0
+  # The next two annotations are used in the policy report generated by the
+  # Audit scanner. Severity indicates policy check result criticality and
+  # Category indicates policy category. See more here at docs.kubewarden.io
+  io.kubewarden.policy.severity: medium
+  io.kubewarden.policy.category: Resource validation
+```
+
+
+**1. Enabling Background Audit Checks:**
+
+The metadata file includes a flag, `backgroundAudit`,  that enables the
+background audit checks for a specific policy. By default, this flag is set to
+"true." The background audit checks feature allows continuous evaluation of
+resources running in the Kubernetes cluster to ensure compliance with the
+deployed policies. When policies are deployed or updated, potentially altering
+the allowed configurations within the cluster, the background audit checks
+assist operators in identifying any violations of security and compliance
+rules. See more at the audit scanner documentation
+
+**2. Defining Kubernetes Resources policies can access:**
+
+Within the metadata file, using the `contextAwareResources` field, users can
+define which Kubernetes resources the policy is allowed to access. For example,
+if the policy need access to `Namespace` resource. The policy author can define
+the `contextAwareResources` as:
+
+```yaml
+[...]
+contextAwareResources:
+  - apiVersion: v1 kind: Namespace
+[...]
+```
+
+**3. Specifying Policy Type: Mutating or Non-Mutating:**
+
+The metadata file contains a flag, `mutating`,  that allows users to designate
+a policy as either a mutating policy or a non-mutating policy. A mutating
+policy modifies the incoming requests or the resources being managed, while a
+non-mutating policy only observes and enforces restrictions without making any
+changes. This distinction is crucial in determining how policies interact with
+the Kubernetes resources and their impact on the cluster.
+
+**4. Defining Resource Type Targets:**
+
+The metadata file provides users with the ability to define the rules within
+the `rules` field, which determine the resource types to which the policy
+applies. This feature empowers users to exercise precise control over policy
+enforcement, guaranteeing that policies are exclusively applied to the intended
+resource types. With this fine-grained control, users can ensure that policies
+are targeted accurately, aligning with their specific requirements and avoiding
+any unintended application to unrelated resource types.
+
+---
+
+The `metadata.yml` file is used by the `kwctl annonate` command to, as the name
```

Comment:
This paragraph should be part of the beginning in my opinion, after the mention to `kwctl scaffold`.